src/components/Home/HomeMap.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import { Menu, Popup, Icon, Label } from 'semantic-ui-react';
+import GoogleMapReact from 'google-map-react';
+
+const style = {
+  borderRadius: 0,
+  border: '1px solid white',
+  opacity: 0.7,
+  backgroundColor: 'grey',
+  color: 'white'
+};
+
+const AnyReactComponent = ({ opened, onClose, labelClass, mouseEnter, mouseLeave, clicked, date, upVote, downVote, city, text, color }) => (
+  <Popup
+    trigger={(
+      <Label
+        as='a'
+        className={labelClass}
+        onClick={clicked}
+        onMouseEnter={mouseEnter}
+        onMouseLeave={mouseLeave}
+        color={color}
+        pointing='below'
+        circular
+      >
+        {text}
+      </Label>
+    )}
+    style={style}
+    size='mini'
+    on='click'
+    open={opened}
+    onClose={onClose}
+  >
+    <Menu.Header>
+      {city}
+    </Menu.Header>
+    <span>Posted: {date}</span>
+    <Menu size='tiny'>
+      <Menu.Item as='a'>
+        <Icon name='thumbs up' />
+        <Label color='teal' floating>
+          {upVote}
+        </Label>
+      </Menu.Item>
+      <Menu.Item as='a'>
+        <Icon name='thumbs down' />
+        <Label color='red' floating>
+          {downVote}
+        </Label>
+      </Menu.Item>
+    </Menu>
+  </Popup>
+);
+
+let today = new Date();
+let dateToday = today.getFullYear() + '-' + (today.getMonth() + 1) + '-' + today.getDate();
```

Comment:
These utility functions need to be placed in a separate file inside like a `/utils` directory. `jsx` files need to ONLY have component related code.

Reply:
Also the naming convention is a little confusing in this one. Also try not to use `+` for strings. Try to use Javascript's ES6 string interpolation. You can do:
```
let currentDate = new Date();
let formattedCurrentDate = `${today.getFullYear()}-${today.getMonth() + 1}-${today.getDate()}`;
```